src/Assets/TestPackages/dotnet-new/test_templates/TemplateWithConditionalParameters/Test.cs

```diff
@@ -2,10 +2,10 @@
 // value of paramA: placeholderA
 // value of paramB: placeholderB
 
-//#if( A_enabled )
+//#if( paramA )
```

Comment:
I think it was correct before - it should show up the values of `A_enabled` and `B_enabled`
`paramA` and `paramB` are string parameters and cannot be used this way.

Reply:
It is reasonable, but on the other hand, it complicates understanding the test:
- maybe it is worth to keep both to check the value of `A_enabled` and `B_enabled` too.
- If you'd like to use `paramA` and `paramB` in condition as the boolean, I suggest tochange `datatype` to boolean.  It is risky: string parameters should not be used in condition as `if (paramA)`, because there is no validation, and user can enter any other string and then condition won't be correct. Or consider putting the comment that it is intentionally incorrect to test it. May also help people who look on tests as the reference.